pip-freeze.txt

```diff
@@ -8,6 +8,7 @@ billiard==3.3.0.22
 boto==2.38.0
 celery==3.1.19
 colorama==0.3.7
+confmodel==0.2.0
```

Comment:
I'm guessing this is already the plan, but just in case, we should ask the relevant people whether they are alright with us including this dependency.